lib/dialogs/share.jsx

```diff
@@ -111,7 +111,7 @@ export default React.createClass( {
 		const { note } = this.props.params;
 		const data = note && note.data || {};
 		const isPublished = includes( data.systemTags, 'published' );
-		const publishURL = isPublished && data.publishURL !== '' && data.publishURL && `http://simp.ly/publish/${data.publishURL}` || null;
+		const publishURL = isPublished && data.publishURL !== '' && data.publishURL && `http://simp.ly/p/${data.publishURL}` || null;
```

Comment:
Now would be a good time to clean up this line. What's that `null` doing there anyway?

How about something like this:
```js
const publishURL = ! isEmpty( data.publishURL ) && data.publishURL;
```

With this, `publishURL` will either be a non-empty string, or it will be `undefined`, both of which probably should work fine for the rest of the file because `if ( ! publishURL )` will still evaluate to `false` and we test for the section stuff with `isPublished`

I did not test this.